mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -933,4 +978,63 @@ class spawn_point_types {
 			};
 		};
 	};
+
+	class boat_light {
+		name = "Light Boats";
+		RESPAWN_MEDIUM;
+
+		class categories {
+			class light_transport_m2 {
+				name = "STAB M2";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "light", "boat", "m2" } };
+			};
+			class light_transport_m60 {
+				name = "STAB M60";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "light", "boat", "m60" } };
+			};
+			class light_transport_mk18 {
+				name = "STAB MK18";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "light", "boat", "mk18" } };
+			};
+		};
+	};
+
+	class boat_large {
+		name = "Large Boats";
+		WRECK_LONG;
+
+		class categories {
+			class large_boat_40mm {
+				name = "Nasty Boat-40mm";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "large", "boat", "40mm" } };
+			};
+			class large_boat_mortar {
+				name = "Nasty Boat-Mortar";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "large", "boat", "mortar" } };
+			};
+		};
+	};
+
+	class boat_small {
+		name = "Small Boats";
+		RESPAWN_SHORT;
```

Comment:
PBRs have more firepower than the STAB rigs, so might make more sense to default them to a longer respawn time?

```suggestion
		RESPAWN_LONG;
```